app/src/main/java/com/example/perfectweatherallyear/model/DayWeather.kt

```diff
@@ -2,7 +2,9 @@ package com.example.perfectweatherallyear.model
 
 
 data class DayWeather(
-    val temperature: String,
-    val precipitation: Int,
-    val wind: Int
+    val temperatureMin: String,
+    val temperatureMax: String,
+    val precipitation: String,
+    val wind: String,
+    val date: String
```

Comment:
date - is not part of the weather. 
Create new data class Day wich will contain field
date: Date
dayWeather: WeatherParameters